src/Compilers/CSharp/Portable/Lowering/StateMachineRewriter/MethodToStateMachineRewriter.cs

```diff
@@ -621,6 +622,52 @@ public override BoundNode VisitBlock(BoundBlock node)
             return PossibleIteratorScope(node.Locals, () => (BoundStatement)base.VisitBlock(node));
         }
 
+        public override BoundNode VisitScope(BoundScope node)
```

Comment:
There's no change to the LambdaRewriter analysis -- I assume that this Scope just naturally gets picked in the analysis when assigning variables to scopes?